### ISSUE TITLE
Injection recovery_time should probably be Duration or TimeUnit types.

### ISSUE THREAD

**Issue by jtyoung84** [AllenNeuralDynamics/aind-data-schema#247]
**Describe the bug**
Currently, the injection class looks like:
```
class Injection(SubjectProcedure):
    ....
    recovery_time: Optional[time] = Field(None, title="Recovery time")
    injection_duration: Optional[float] = Field(None, title="Injection duration")
    injection_duration_unit: Optional[TimeUnit] = Field(TimeUnit.M, title="Injection duration unit")
...
```

**To Reproduce**
1. Install the aind_data_schema package
2. Construct an Injection instance, for example:
```
from datetime import datetime
from aind_data_schema.procedures import Injection
inj=Injection(recovery_time=85.0, start_date=datetime.now().date(), end_date=datetime.now().date(), experimenter_full_name="someone", protocol_id="12345")
>>> print(inj.recovery_time)
00:01:25

```
3. Check the recovery_time:
```
print(inj.recovery_time)
'00:01:25'
```

**Expected behavior**
The recovery_time is represented as a time of day. It should probably be a duration?

**Additional context**
The Recovery Time in NSB's sharepoint database looks like "30"

**Comment by jtyoung84** [comment]
Try to find other ones...

**Comment by saskiad** [comment]
closed with PR #260
